src/rust/generator-dispatcher/tests/generator_dispatcher_integration_tests.rs

```diff
@@ -56,7 +56,50 @@ async fn test_dispatcher_inserts_job_into_plugin_work_queue(
     };
 
     // Send in the Raw Log Event
-    let log_event: Bytes = r#"i am some sort of raw log event"#.into();
+    let log_event: Bytes = r#"
```

Comment:
could we perhaps stop copy pasting this log event :pray:

should be easy to e.g. `include_bytes!()` from a file or something

Reply:
I don't think we should. If we `include_bytes!()` it's easy to fundamentally alter the test invariants "at a distance" by changing _that file_. This would cause the assertions a test makes to change (possibly radically) without a code change which obviously impacts that test.

Also, changing that file would impact _multiple tests_ simultaneously, which seems difficult to reason about. While this event _is_ copy-pasta'd everywhere, and therefore _looks_ the same everywhere, I'm not sure it actually _is the same_.